### ISSUE TITLE
Feature Request: Ability to remove Gravebot messages after a set amount of time

### ISSUE THREAD

**Issue by GrayBiz** [Gravestorm/Gravebot#128]
We are loving everything Gravebot can do, but sometimes the members of our chat groups get a little overzealous with Gravebot's chat commands. We've noticed that the RhinoBot music chat bot will remove certain messages after a minute or so if they are unnecessary to the chat as a whole. It might be nice to have some way to set it so that Gravebot will automatically delete its own posts after a set amount of time, say 5 minutes.

**Comment by dustinblackman** [comment]
You're completely right. Would you prefer it removing all messages after a specified time, or maybe just specifics?

**Comment by GrayBiz** [comment]
Well, there definitely may be some situations where you may not want to remove posts, like say with the !meme command, however even then there could be possibility for spam. With that said, I think the best solution probably would be to have all messages be removed, since the command to generate whatever the user found can always be used again, or they could save the link that's generated if they want to keep it.

This leads to another possibility of setting up the option to simply replace the picture with a link instead of embedding the picture. I'm not sure how possible this is since I know that Discord will auto embed images from links unless you have that permission disabled, but maybe there is a markdown workaround?

Another thing that may improve the auto-delete functionality would be to have the time period set based on how many times Gravebot generates a new post. So, if the users in the chat issue only 1 command in say 5 minutes, then it will delete the image at the end of the period, but if the users are calling on it constantly, the posts could then be set to be deleted after only 1 minute.
